### ISSUE TITLE
UX issues with existing selectors

### ISSUE THREAD

**Issue by NickIliev** [NativeScript/theme#90]
While applying the theme in [https://github.com/NativeScript/nativescript-sdk-examples-ng](https://github.com/NativeScript/nativescript-sdk-examples-ng) I have come across some UX and programmatic issues.
Here is some of them in order of appearance:
- [ ] ListView items default height is not 48 (its bigger approx 56)
- [x] Can not set margin and padding 16 which is a must for Android (only values 10,15,20 .. and so on)
- [ ] Style for rounded buttons and active buttons not working at this point

```
<Button class="btn btn-rounded-sm" text="Small rounded corners"></Button>
<Button class="btn btn-rounded-lg" text="Large rounded corners"></Button>
<Button class="btn btn-active" text="I’m highlighted when tapped"></Button>
```
- [ ] applying margin on Button won't work when parent is Stack
  Example: 

```
<StackLayout>
    <Button class="btn btn-primary m-15" text="Display Action Dialog" (tap)="displayActionDialog()">     </Button>
</StackLayout>
```

**Comment by NathanWalker** [comment]
@NickIliev thanks for reporting, this will be fixed by 1.0 and we can add multiples of 8 to the padding/margin sets

**Comment by NickIliev** [comment]
I will add more issues in this thread which I am encountering during the integration of the theme in the sample app.

**Comment by NathanWalker** [comment]
@NickIliev multiples of 4 have been added to the utility classes providing 4 (m-4, etc.), 8, 12, 16, 20, 24, 28. Published with `0.1.3` available now.

@TheOriginalJosh you mind taking a look at the btn-rounded issue mentioned above?

@NickIliev using margin with parent as stack...hmm, why is that the case? I tested and noticed the same buy why?

**Comment by NickIliev** [comment]
@NathanWalker agreed on the matter with the stack.

However I have one problem when styling bigger margins.
For example following the [Material Design guide the navigation drawers](https://material.google.com/patterns/navigation-drawer.html#navigation-drawer-specs) should have high margin 56dp. However try gin to set margin like
`m-r-55 or m-r-56 ` won't work at all.
In the same time m-r-15 and m-r-16 will work as expected.
Any reasons why bigger making are not applied. 
I've noticed that you guys have added styles for side-drawers. Probably the default right margin can be applied with class="sidedrawer-left" !?
